python/cucim/src/cucim/skimage/data/_binary_blobs.py

```diff
@@ -1,8 +1,14 @@
 import cupy as cp
 
+from .._shared.filters import gaussian
+from .._shared.utils import deprecate_kwarg
 
+
+@deprecate_kwarg(
+    {"seed": "rng"}, deprecated_version="23.12.00", removed_version="24.12.00"
+)
 def binary_blobs(
-    length=512, blob_size_fraction=0.1, n_dim=2, volume_fraction=0.5, seed=None
+    length=512, blob_size_fraction=0.1, n_dim=2, volume_fraction=0.5, rng=None
 ):
```

Comment:
Does this just forward the `seed` argument to `rng` and raise a depreciation warning?

Reply:
Yes, it forwards and raises a `FutureWarning` (the decorator is the same one used by scikit-image itself).

I think scikit-image changed seed->rng before releasing 0.21, though so they wouldn't have the dual `deprecate_kwarg` decorators that are unfortunately present a couple places here.